README.md

```diff
@@ -16,8 +16,6 @@
 	* [Error Handling](#error-handling)
 	* [Access Modifiers](#access-modifiers)
 	* [Imports](#imports)
-	* [Type Declarations](#type-declarations)
-	* [Type Inference](#type-inference)
```

Comment:
Yeah they are now subsections of type so I figured they'd be better off the toc, otherwise we need one more tab